sdk/storage/azure-storage-blob/azure/storage/blob/aio/_container_client_async.py

```diff
@@ -1256,7 +1256,7 @@ async def delete_blobs(
     @distributed_trace_async
     async def set_standard_blob_tier_blobs(
         self, standard_blob_tier: Union[str, 'StandardBlobTier'],
-        *blobs: Union[str, Dict[str, Any], BlobProperties],
```

Comment:
What's the `Dict[str, Any]` that we were supporting here? Is there any chance this might also have similar issues with ignore snapshot/version data?

Reply:
This input is an entry per blob where you can specify snapshot, version, etc. It is essentially a paired down BlobProperties if you don't already have one. We do support all "problematic" things here by design. The structure of this dictionary is well documented in the docstring.
https://github.com/Azure/azure-sdk-for-python/blob/674698cd5048b5625713d73406c99681df585803/sdk/storage/azure-storage-blob/azure/storage/blob/_container_client.py#L1574-L1581